### PR TITLE
[Pages] Gatsby framework guide style guide review

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

**Review by kristianfreeman — APPROVED** [review]

**Review by lukeed — APPROVED** [review]

File: products/pages/src/content/framework-guides/deploy-a-gatsby-site.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ $ git push -u origin main
 
 ## Deploying with Cloudflare Pages
 
-You can deploy your site to Cloudflare Pages by going to the dashboard and creating a new site. Select your new GitHub repository that you created above, and in the configuration section, provide the following information:
+Deploy your site to Pages by logging into [Cloudflare dashboard](https://dash.cloudflare.com/) > going to the Account Home > Pages dashboard and selecting **Create a project**. Select the new GitHub repository that you created and, in the **Set up builds and deployments** section, provide the following information:
```
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/deploy-a-gatsby-site.md]:
```suggestion
Deploy your site to Pages by logging into [Cloudflare dashboard](https://dash.cloudflare.com/) > Account Home > **Pages** dashboard and selecting **Create a project**. Select the new GitHub repository that you created and, in the **Set up builds and deployments** section, provide the following information:
```

File: products/pages/src/content/framework-guides/deploy-a-gatsby-site.md [hunk 2]
```diff
@@ -46,16 +46,17 @@ You can deploy your site to Cloudflare Pages by going to the dashboard and creat
 
 </TableLayout>
 
-Once you've configured your site, you can begin your first deploy. You should see Cloudflare Pages installing `gatsby`, your project dependencies, and building your site, before deploying it.
+After configuring your site, you can begin your first deploy. You should see Cloudflare Pages installing `gatsby`, your project dependencies, and building your site, before deploying it.
 
 <Aside>
```
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/deploy-a-gatsby-site.md]:
```suggestion
<Aside type="note">
```

File: products/pages/src/content/framework-guides/deploy-a-gatsby-site.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ $ git push -u origin main
 
 ## Deploying with Cloudflare Pages
 
-You can deploy your site to Cloudflare Pages by going to the dashboard and creating a new site. Select your new GitHub repository that you created above, and in the configuration section, provide the following information:
+Deploy your site to Pages by logging into [Cloudflare dashboard](https://dash.cloudflare.com/) > Account Home > **Pages** dashboard and selecting **Create a project**. Select the new GitHub repository that you created and, in the **Set up builds and deployments** section, provide the following information:
```
Comment (marciocloudflare) [inline products/pages/src/content/framework-guides/deploy-a-gatsby-site.md]: I think `Account Home` should be in bold

**Review by marciocloudflare — APPROVED** [review]

File: products/pages/src/content/framework-guides/deploy-a-gatsby-site.md (hunk 3 above)
Comment (deadlypants1973) [inline products/pages/src/content/framework-guides/deploy-a-gatsby-site.md]:
```suggestion
Deploy your site to Pages by logging into [Cloudflare dashboard](https://dash.cloudflare.com/) > **Account Home** > **Pages** dashboard and selecting **Create a project**. Select the new GitHub repository that you created and, in the **Set up builds and deployments** section, provide the following information:
```

**Pull request merged**